### ISSUE TITLE
Batch_norm W, b gradients for RNN0 layer are None

### ISSUE THREAD

**Issue by SiddGururani** [SeanNaren/deepspeech.pytorch#75]
I was in the process of adding Tensorboard logging to the training process when I noticed that some of the gradients were None which led me to investigate this further.

To investigate, I passed a single batch of data and ran a backward pass. I then iterate over the named parameters and check for any None gradients. In doing so, I found this: 
``` 
module.rnns.0.batch_norm.module.weight True
module.rnns.0.batch_norm.module.bias True 
```
True indicates that they have require_grad set to True.

It doesn't seem like these gradients are meant to be zero so I've raised an issue. Can someone explain to me why this would be the case?

**Comment by ryanleary** [comment]
Do you have a fork with the tensorboard integration @SiddGururani? I was looking into doing something similar, and would love to not reinvent the wheel.

**Comment by SiddGururani** [comment]
I don't, unfortunately. I cloned the repo and didn't think I was going to need to make changes. I'll fork and push the changes for tensorboard integration today.

**Comment by ryanleary** [comment]
All you would have to do is hit the fork button, then where you have the repo cloned (with your changes) run `git remote add sidd git@github.com:SiddGururani/deepspeech.pytorch.git` (you can replace `sidd` with whatever you like`. Then you can push your changes to your own fork.

**Comment by ryanleary** [comment]
This is a strange artifact of pytorch registering parameters that shouldn't be there (never called in the forward function). @SeanNaren pushed up a fix which should fix it.
